blobmodel/blobs.py

```diff
@@ -34,6 +36,12 @@ def __init__(
         self.pos_y = pos_y
         self.t_init = t_init
         self.t_drain = t_drain
+        self.prop_shape_parameters = (
```

Comment:
Yeah, as for now only "lam" is implemented, but wanted to leave it open in case we want to have other parameters, even though there is such thing planned yet.